src/others/observer/lv_observer.c

```diff
@@ -452,6 +452,11 @@ lv_observer_t * lv_obj_bind_checked(lv_obj_t * obj, lv_subject_t * subject)
 #if LV_USE_LABEL
 lv_observer_t * lv_label_bind_text(lv_obj_t * obj, lv_subject_t * subject, const char * fmt)
 {
+    if((subject == NULL) || (obj == NULL)) {
+        LV_LOG_WARN("NULL pointer");
+        return NULL;
```

Comment:
It is recommended to use `LV_ASSERT_NULL`

Reply:
I didn't initially use the LV_ASSERT_NULL macro because it looks like the routines are allowed to return NULL and the macro will block execution instead of returning NULL. I can see where it might be a matter of preference on whether the routine blocked when it received invalid input or just returned NULL. Since there were already other checks in the routine that returned NULL, I implemented it to also return NULL. But I think using the macro would be reasonable as well.